### PR TITLE
support marked ^2.0.0

### PR DESCRIPTION
**Pull request by UziTech**
Support marked ^2.0.0 as peer dependency

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
   "author": "Mikael Brevik",
   "license": "MIT",
   "peerDependencies": {
-    "marked": "^1.0.0"
+    "marked": "^1.0.0 || ^2.0.0"
```
Comment (vansergen) [inline package.json:22]: Why do you want to support `marked@1`? It has some [vulnerabilities](https://github.com/advisories/GHSA-4r62-v4vq-hr96)

Comment (UziTech) [reply package.json:22]: This package still works with v1. If someone needs to use v1 for whatever reasons they should not be prevented from updating this package.

**Comment by brettz9** [comment]
Is this ok to merge and release then? Thanks!

**Comment by thetutlage** [comment]
Any chance of getting this merged and released?

**Review by RomainLanz — APPROVED** [review]

**Comment by ktalebian** [comment]
@RomainLanz @UziTech can you take a look at the CI failure and hopefully get this merged?

**Comment by RomainLanz** [comment]
The CI error is not directly related to this PR.
The error happens because the Travis config is outdated and use an EOL Node.js version.

There's already a PR to change it https://github.com/mikaelbr/marked-terminal/pull/65

**Comment by ktalebian** [comment]
@RomainLanz ah makes sense. So plan is to merge https://github.com/mikaelbr/marked-terminal/pull/65 and then this PR. 

When do you reckon that could happen? Are you the one who is going to do the merge and release or that's someone else?

**Comment by RomainLanz** [comment]
Hey @ktalebian! 👋 

I believe there's a misconception. 😄 
I'm also waiting on the merge and the release, I'm not related to this package or repository at all.

**Comment by UziTech** [comment]
@mikaelbr can this be merged?

**Pull request merged**